src/components/Book/Book.js

```diff
@@ -35,10 +45,15 @@ function Book(props) {
     </div>
   );
 }
+// Book.defaultProps = {
+//   categeory: 'Action',
+// };
+
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

Reply:
Thank you for your suggestion.
Removed the same from local, and same will reflect in next PR.